src/getCommands.js

```diff
@@ -8,8 +9,14 @@ const findPlugins = require('./findPlugins');
  * @param  {String}       name Name of the plugin
  * @return {Array|Object}      Plugin's commands
  */
-const getPluginCommands = cwd => name =>
-  require(path.join(cwd, 'node_modules', name));
+const getPluginCommands = cwd => name => {
+  const nested = path.join(cwd, 'node_modules', name);
+
+  /* For rnpm installed locally with flat dependencies structure */
+  const flat = path.join(cwd, '..', name);
```

Comment:
To clarify edge-case, when installed globally and required plugin found in appRoot.
- look in node_modules/rnpm
- look in global node_modules
- throw (always)

We have to change it to:
- **always** look in project/node_modules/